src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -211,11 +211,13 @@ .. conflictResolution.RelatedLocations
 
                 if (IsRenameValid(conflictResolution, renamedSymbolInNewSolution))
                 {
+                    var declarationDocument = await conflictResolution.CurrentSolution.GetRequiredDocumentAsync(_documentIdOfRenameSymbolDeclaration, includeSourceGenerated: true).ConfigureAwait(false);
```

Comment:
so at these locations int he code, i would have expected we had a renameoptions somewhere.  and we would use that to pass the right value along for includeSourceGenerated.

Reply:
Yeah, that's fair, I can plumb it through to all of these places. I went on the theory that there would only be locations in generated files if someone had asked for them, but at this point the code is no more efficient whether true or false is passed in. Can certainly see how making that stricter is appealing, in case there are bugs at higher layers.